cypress_buildspec.yaml

```diff
@@ -0,0 +1,44 @@
+version: 0.2
+env:
+  shell: bash
+  secrets-manager:
+    SONAR_TOKEN: arn:aws:secretsmanager:eu-central-1:911845998067:secret:SonarCloud-7370vM:token
```

Comment:
@maurizioflauti i think this token is not used in this builder, delete it

Reply:
ok